src/components/SocialMediaLinks/SocialMediaLinks.stories.tsx

```diff
@@ -0,0 +1,249 @@
+import { withDesign } from "storybook-addon-designs";
+import { layoutTypesArray } from "../../helpers/types";
+import { Box, VStack } from "@chakra-ui/react";
+import Heading from "../Heading/Heading";
+import { Source } from "@storybook/blocks";
+
+const meta: Meta<typeof SocialMediaLinks> = {
+  title: "Components/Navigation/SocialMediaLinks",
+  component: SocialMediaLinks,
+  decorators: [withDesign],
+  argTypes: {
+    borders: {
+      table: { disable: true },
+    },
+    className: { table: { disable: true } },
+    color: {
+      table: { disable: true },
+    },
+    id: { table: { disable: true } },
+    layout: {
+      table: { disable: true },
+    },
+    linksData: { table: { disable: true } },
+    showLabels: { table: { disable: true } },
+    size: {
+      table: { disable: true },
+    },
+  },
+  parameters: {
+    controls: {
+      hideNoControlsWarning: true,
+    },
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=77103-126004&t=Yx1GyAfBJyhuGIGs-4",
+    },
+    jest: ["SocialMediaLinks.test.tsx"],
+  },
+};
+export default meta;
+type Story = StoryObj<typeof SocialMediaLinks>;
+
+export const Default: Story = {};
+
+export const WithControls: Story = {
+  args: {
+    borders: "none",
+    color: "textDefault",
+    className: undefined,
+    id: undefined,
+    layout: "row",
+    linksData: [
+      { type: "facebook" },
+      {
+        type: "twitter",
+        url: "twitter.com/elsewhere",
+        labelText: "Alt Twitter",
+      },
+      {
+        type: "instagram",
+      },
+    ],
+    showLabels: true,
+    size: "small",
+  },
+  argTypes: {
+    borders: {
+      control: { type: "radio" },
+      options: borderTypeArray,
+      table: {
+        disable: false,
+        defaultValue: { summary: "none" },
+      },
+    },
+    className: {
+      control: true,
+      table: {
+        disable: false,
+      },
+    },
+    color: {
+      control: { type: "radio" },
+      options: colorTypeArray,
+      table: {
+        disable: false,
+        defaultValue: { summary: "textDefault" },
+      },
+    },
+    id: {
+      control: true,
+      table: {
+        disable: false,
+      },
+    },
+    layout: {
+      control: { type: "radio" },
+      options: layoutTypesArray,
+      table: {
+        disable: false,
+        defaultValue: { summary: "row" },
+      },
+    },
+    linksData: { table: { disable: false } },
+    showLabels: {
+      table: {
+        disable: false,
+        defaultValue: { summary: false },
+      },
+    },
+    size: {
+      control: { type: "radio" },
+      options: sizeTypeArray,
+      table: {
+        disable: false,
+        defaultValue: { summary: "small" },
+      },
+    },
+  },
+};
+
+export const LayoutVariations: Story = {
+  render: () => (
+    <VStack align="stretch" spacing="l">
+      <Heading id="heading-layout" level="three" text="Layout Variations" />
```

Comment:
For consistency with other story docs, this heading should be pulled out of here and rendered using `mdx`.  This would apply to the formatting of all `Story` elements.